tools/demo/react-native/index.ts

```diff
@@ -1,21 +1,27 @@
 import { DemoSharedBase } from '../utils';
-import { ReactNative } from '@ammarahm-ed/react-native';
-
-const RNBridge = new ReactNative();
+//@ts-ignore
+import Auth0 from 'react-native-auth0';
```

Comment:
Just leaving this review comment to remind us to drop mentions of `react-native-auth0` when we come to merge this PR.

Incidentally, I think TypeScript will forgive `require()` imports for lacking typings:

```suggestion
const Auth0 = require('react-native-auth0');
```

... but if you do apply this code suggestion, don't let GitHub mark this review comment as resolved, as it's reminding us to remove `react-native-auth0` 😅

Reply:
Resolving this, as deintegrating `react-native-auth0` is firmly on our to-do list.